iron-dropdown.html

```diff
@@ -418,7 +421,16 @@
             return;
           }
 
-          this.style[this.horizontalAlign] =
+          // In RTL, the direction flips, so what is "right" in LTR becomes "left".
+          var isRTL = window.getComputedStyle(this).direction == 'rtl';
+          var actualHorizontalAlign;
```

Comment:
What do you think about moving this a read-only accessor called `localeHorizontalAlign` or similar?